Interpolation_Search/README.md

```diff
@@ -0,0 +1,65 @@
+#  Interpolation Search
+
+Interpolation sort is an improvement over binary search.  
+Binary search always checks the value at the middle index. 
+But interpolation search may check at the different locations based on the value of the element being searched.  
+For interpolation search to work efficiently the array elements/data should be sorted and uniformly distributed. 
+
+## Algorithm
+
+Step 1: Make start = 0 & end = n-1<br />
+Step 2: Calculate position(pos) to start searching at:<br /> 
+`pos = start + ((end - start) / (Array[end] - Array[start])) * (element - Array[start])`<br />
+Step 3: If Array[pos] == element, element found at the index pos.<br />
+Step 4: Otherwise if element > Array[pos] make start = pos + 1 and repeat the same process <br />
+Step 5: Else if element < Array[pos] make end = pos-1 and repeat the same process.
+
+## Example
+
+Find element : 4 
+
+- Iteration 1 <br />
+![Screenshot (162)](https://user-images.githubusercontent.com/28682701/58583702-3d6dcf00-8272-11e9-98c2-caea66db889b.png)
+
+Array[1] < 4 hence start = 1 + 1 = 2 
+
+- Iteration 2 <br />
+![Screenshot (163)](https://user-images.githubusercontent.com/28682701/58584706-84f55a80-8274-11e9-99ee-47d62e591e73.png)
+
+## Pseudocode
+```
+A → Array list
+N → Size of A
+X → Target Value
+
+Procedure Interpolation_Search()
+
+   Set Lo  →  0
```

Comment:
Fix the indentation. 4 space, 8 space & so on...

Reply:
@marwahmanbir Done